optional/postgresql/Dockerfile

```diff
@@ -1,12 +1,12 @@
-ARG DISTRO=alpine:3.10
+ARG DISTRO=alpine:3.12
```

Comment:
The upgrade from Alpine 3.10 to 3.12 resulted in an upgrade from PostgreSQL 11.7 to 12.3: 
* https://alpine.pkgs.org/3.10/alpine-main-x86_64/postgresql-11.7-r0.apk.html
* https://alpine.pkgs.org/3.12/alpine-main-x86_64/postgresql-12.3-r2.apk.html

Existing installations require a manual upgrade of the PostgreSQL instance: 
* https://www.postgresql.org/docs/current/pgupgrade.html

The following project has helped me a lot:
* https://github.com/tianon/docker-postgres-upgrade
* https://github.com/tianon/docker-postgres-upgrade/tree/master/11-to-12

For example, I have executed the following commands:
```
docker-compose -p mailu down
mv /data/mailu/psql_db /data/mailu/psql_tmp
mkdir /data/mailu/psql_db
chown 70:70 /data/mailu/psql_db
docker run --rm \
	-v /data/mailu/psql_tmp:/var/lib/postgresql/11/data \
	-v /data/mailu/psql_db:/var/lib/postgresql/12/data \
	tianon/postgres-upgrade:11-to-12
docker-compose -p mailu up -d
rm -rf /data/mailu/psql_tmp
```

Maybe we should document this process somewhere?

/cc @liquidat

Reply:
If I remember correctly, last time we upgraded the alpine base image, we discussed this to and documented that `administrator action is needed` in the release notes of the next version (which was 1.7 probably? :D), we could copy that to the release notes of 1.8 when it's done